compiler/rustc_typeck/src/check/method/confirm.rs

```diff
@@ -81,11 +81,25 @@ impl<'a, 'tcx> ConfirmContext<'a, 'tcx> {
         let rcvr_substs = self.fresh_receiver_substs(self_ty, &pick);
         let all_substs = self.instantiate_method_substs(&pick, segment, rcvr_substs);
 
-        debug!("all_substs={:?}", all_substs);
+        debug!("rcvr_substs={rcvr_substs:?}, all_substs={all_substs:?}");
```

Comment:
Isn't this the same as

```
debug!(?rcvr_substs, ?all_substs);
```

?

Reply:
yeah, you're right. not super familiar with the tracing macros, sorry😅